### PR TITLE
MC-8497: list clusters doc

### PR DESCRIPTION
**Pull request by jasondias9**

### CODE REVIEW

File: source/includes/gcp/_clusters.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+------- | -----------
+`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format
+`nodeCount`<br/>*number* | Number of nodes in the cluster
+`endpoint`<br/>*string* | The IP address of the master node of the cluster. All interactions with the Kubernetes API are done through the master node.
+`location` <br/> *string* | The region zone in which the cluster is running. For regional clusters the nodes of your cluster may span multiple zones within the region.  
+`status` <br/> *string* | The status of the cluster
+`id` <br/> *string* | The cluster is uniquely identfied by the project name, location and cluster name.
+`name` <br/> *string* | The name of the cluster
+
+<!-------------------- RETRIEVE A CLUSTERS -------------------->
```
Comment (simongodard) [inline source/includes/gcp/_clusters.md]: no S needed

File: source/includes/gcp/_clusters.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+`nodeCount`<br/>*number* | Number of nodes in the cluster
+`endpoint`<br/>*string* | The IP address of the master node of the cluster. All interactions with the Kubernetes API are done through the master node.
+`location` <br/> *string* | The region zone in which the cluster is running. For regional clusters the nodes of your cluster may span multiple zones within the region.  
+`status` <br/> *string* | The status of the cluster
+`id` <br/> *string* | The cluster is uniquely identfied by the project name, location and cluster name.
+`name` <br/> *string* | The name of the cluster
+
+<!-------------------- RETRIEVE A CLUSTER -------------------->
+
+#### Retrieve a clusters
```
Comment (SeventhOfNine) [inline source/includes/gcp/_clusters.md]: no 's' in cluster here.

**Review by SeventhOfNine — APPROVED** [review]
Looks good

File: source/includes/gcp/_clusters.md (hunk 2 above)
Comment (SeventhOfNine) [reply source/includes/gcp/_clusters.md]: Fixed

**Pull request merged**